main.py

```diff
@@ -1,11 +1,47 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player_name, player_data in data.items():
+        guild_data = player_data.get("guild")
+        guild = None
+        if guild_data:
+            guild_name = guild_data.get("name")
+            guild_description = guild_data.get("description")
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_name,
+                defaults={"description": guild_description}
+            )
+
+        race_data = player_data["race"]
+        race_name = race_data["name"]
+        race_description = race_data.get("description", "")
+        race, _ = Race.objects.get_or_create(
+            name=race_name,
+            defaults={"description": race_description}
+        )
 
+        skills = race_data.get("skills", [])
```

Comment:
Ensure that 'skills' is always a list in the JSON data to avoid potential errors. Consider adding a default empty list in the `get` method: `race_data.get('skills', [])`.